src/5.0_to_5.1/migrateFilters.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Wraps the MDX of each filter in `filters`.
+ * Mutates `filters`.
+ */
+export function migrateFilters(filters: any[] | undefined): void {
```

Comment:
Typing this as `any` is the downside of the choice of going for mutations everywhere, right?
Can we add a comment explaining why we're using `any`?

Reply:
Yes it it is the downside of the mutations indeed.
I think something btter than `any` could be to type the input as `Mdx[]`, and then do `ts-expect-error` when mutating.
It works realtively well for small logic like this where it's basically a one-liner, however it's more annoying when the logic grows bigger: it's annoying to have to call call `ts-expect-error` for every single mutation.